laws/src/main/scala/cats/laws/ApplicativeLaws.scala

```diff
@@ -7,7 +7,9 @@ import cats.syntax.functor._
 /**
  * Laws that must be obeyed by any [[Applicative]].
  */
-class ApplicativeLaws[F[_]](implicit F: Applicative[F]) extends ApplyLaws[F] {
+trait ApplicativeLaws[F[_]] extends ApplyLaws[F] {
+  implicit override def F: Applicative[F]
```

Comment:
Yes. Syntax in the laws (e.g. `fa.apply`) wouldn't work if there isn't an `implicit Applicative[F]`.